### PR TITLE
[HttpFoundation] Add HTTP_EARLY_HINTS const

### PR DESCRIPTION
**Pull request by ahmadmayahi**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| License       | MIT

### CODE REVIEW

File: src/Symfony/Component/HttpFoundation/Response.php [hunk 1]
```diff
@@ -21,6 +21,7 @@ class Response
     const HTTP_CONTINUE = 100;
     const HTTP_SWITCHING_PROTOCOLS = 101;
     const HTTP_PROCESSING = 102;            // RFC2518
+    const HTTP_EARLY_HINTS = 103;           // RFC 8297
```
Comment (lyrixx) [inline src/Symfony/Component/HttpFoundation/Response.php]: Could you update `RFC 8297` -> `RFC8297` to be consistant with other comments ?

**Review by lyrixx — CHANGES_REQUESTED** [review]

**Review by lyrixx — APPROVED** [review]
Thanks @ahmadmayahi  for your first contribution

:+1:

**Review by nicolas-grekas — APPROVED** [review]
(for 2.7)

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @ahmadmayahi.

**Pull request merged**